### PR TITLE
[14.0][ADD] rma_refund_reason

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**

### CODE REVIEW

File: rma_refund_reason/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "category": "RMA",
+    "author": "ForgeFlow S.L.",
+    "website": "https://github.com/ForgeFlow/stock-rma",
+    "license": "AGPL-3",
+    "depends": ["rma_account", "account_invoice_refund_reason"],
+    "data": [
+        "views/rma_operation_view.xml",
+        "views/rma_order_line_view.xml",
+    ],
+    "demo": [],
```
Comment (AaronHForgeFlow) [inline rma_refund_reason/__manifest__.py]: empty sections can be removed

File: rma_refund_reason/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Add refund reason on rma refund creation process
```
Comment (AaronHForgeFlow) [inline rma_refund_reason/readme/DESCRIPTION.rst:1]: Perhaps saying also that they are configurable in the RMA operations?

File: rma_refund_reason/__init__.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Copyright (C) 2023 ForgeFlow S.L.
```
Comment (AaronHForgeFlow) [inline rma_refund_reason/__init__.py]: Copyright in init files are not needed

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM, only minor comments

File: rma_refund_reason/__manifest__.py (hunk 1 above)
Comment (ChrisOForgeFlow) [reply rma_refund_reason/__manifest__.py]: Done!

File: rma_refund_reason/readme/DESCRIPTION.rst (hunk 2 above)
Comment (ChrisOForgeFlow) [reply rma_refund_reason/readme/DESCRIPTION.rst:1]: Done!

File: rma_refund_reason/__init__.py (hunk 3 above)
Comment (ChrisOForgeFlow) [reply rma_refund_reason/__init__.py]: Done!

**Review by AaronHForgeFlow — APPROVED** [review]
Ok to merge

**Review by mariadforgeflow — APPROVED** [review]
LGTM!

**Pull request merged**